core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1416,5 +1416,8 @@ void preparedTxAlreadyExists(RecoveryAwareTransaction previous,
    @LogMessage(level = WARN)
    @Message(value = "Received unsolicited state from node %s for segment %d of cache %s", id = 396)
    void ignoringUnsolicitedState(Address node, int segment, String cacheName);
+
+   @Message(value = "Cannot enable '%s' in invalidation caches!", id = 397)
```

Comment:
You cannot introduce a logging ID which conflicts with one used by future versions. You should use id 420.